analyzer/ocr/ocr.go

```diff
@@ -76,7 +76,7 @@ func (a *Analyzer) pdfToText(jobID string, content []byte, threshold int) (strin
 		if image, _ := doc.Image(i); image != nil {
 			ocrPages++
 			var img = new(bytes.Buffer)
-			if err := jpeg.Encode(img, image, &jpeg.Options{Quality: 50}); err != nil {
```

Comment:
I wonder if this was the issue we were experiencing? Although if it works with `png` I don't see any reason to change to jpeg. From what I can tell `jpeg` is lossy whereas `png` is lossless so that could very well explain why we ran into the failure?

Reply:
Actually, reading https://www.labnol.org/software/tutorials/jpeg-vs-png-image-quality-or-bandwidth/5385/ it appears that `png` is better for images containing text.

Reply:
Even with PNG, theres a word in one of the tests that tesseract doesnt recognize on Travis but does recognize on my laptop. I've changed the assertion for now, but it's in this commit: https://github.com/RTradeLtd/Lens/pull/11/commits/e97eae1ce8a852b55c5f455d38da777212ecb9ed